modules/ztp-single-node-clusters.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-single-node-clusters_{context}"]
+
+= Single node clusters
+
+You can use ZTP to deploy single node clusters to run distributed units (DUs) on small hardware footprints at disconnected far edge sites.
+A single node cluster runs {product-title} Container Platform on top of one bare-metal machine, hence the single node.
+Single edge node servers contain a single node with supervisor functions and worker functions on the same host that are deployed at low
+bandwidth or disconnected edge sites.
+
+Hosts are partitioned to isolate the platform services, housekeeping tasks, and infrastructure pods to run on a reserved set of CPUs.
+This is useful for resource-constrained environments, such as single-node production deployments, where you want to reserve most of the
```

Comment:
sugg: s/This/Workload partitioning/ because there is a lot going on in the preceding sentence.

Reply:
Comment accepted.